scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -259,6 +259,7 @@ SG_ PRECHARGE_ERROR : 10|1@1+ (1,0) [0|1] "" DEBUG
 BO_ 110 BMS_CHARGER: 1 BMS
 SG_ Is_Connected : 0|1@1+ (1,0) [0|1] "" DEBUG
 SG_ IS_CHARGING_COMPLETE : 1|1@1+ (1,0) [0|1] "" DEBUG
+SG_ IS_CHARGING_ENABLED : 2|1@1+ (1,0) [0|1] "" BMS
```

Comment:
```suggestion
SG_ CHARGING_STATUS : 2|1@1+ (1,0) [0|1] "" BMS
```
to make it more user friendly. i'm not totally sold on this name

Reply:
What do we think of CHARGE_SWITCH?